docs/linux/sql-server-linux-active-directory-authentication.md

```diff
@@ -96,19 +89,12 @@ Use the following steps to join a [!INCLUDE[ssNoVersion](../includes/ssnoversion
 
    - **RHEL**:
 
-     Edit the `/etc/sysconfig/network-scripts/ifcfg-eth0` file (or other interface config file as appropriate) so that your AD domain controller's IP address is listed as a DNS server:
-
-     ```/etc/sysconfig/network-scripts/ifcfg-eth0
-     <...>
-     PEERDNS=no
-     DNS1=**<AD domain controller IP address>**
-     ```
-
-     After editing this file, restart the network service:
-
-     ```bash
-     sudo systemctl restart network
-     ```
+     Insure there are no DNS1 DNS2 or PEERDNS entries specified in `/etc/sysconfig/network-scripts/ifcfg-eth*` files
```

Comment:
Same comment as above. This is incorrect.

Reply:
The dns resolver is set in resolv.conf. Setting it in ifcfg files is just needless duplication.

Reply:
Same comment as above